isis/tests/cnetbin2pvlTests.cpp

```diff
@@ -0,0 +1,35 @@
+#include <QTemporaryDir>
+#include <iostream>
+
+#include "Pvl.h"
+#include "cnetbin2pvl.h"
+#include "Fixtures.h"
+#include "Progress.h"
+#include "gmock/gmock.h"
+#include "FileName.h"
+#include "UserInterface.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cnetbin2pvl.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestCnetbin2pvlDefault) {
+  QTemporaryDir prefix;
+  ASSERT_TRUE(prefix.isValid());
+
+  QString pvlOut = prefix.path()+"/cnetbin2pvlNetwork.pvl";
+
+  QVector<QString> args = {"to="+pvlOut};
+  UserInterface ui(APP_XML, args);
+
+  Progress progress;
+  cnetbin2pvl(*network, ui, &progress);
+
+  Pvl pvl;
+  try {
+    pvl.read(pvlOut);
```

Comment:
Binary control networks have a PVL formatted header, so this will still pass even if it writes out a binary network.

You should add an additional check that the PVL contains an object called ControlPoint